float64 steps
int64 input_dim
int64 state_dim

float64 total_cost


mppi_ros/Array cost_vector
mppi_ros/Array time_vector
mppi_ros/Array[] state_vector
mppi_ros/Array[] input_vector
mppi_ros/Array[] noise_vector
